hardware/raspberrypi/bootmodes/net_tutorial.md

```diff
@@ -35,6 +35,22 @@ Ensure the output `0x3020000a` is correct.
 
 The client configuration is almost done. The final thing to do is to remove the `program_usb_boot_mode` line from `config.txt` (make sure there is no blank line at the end). You can do this with `sudo nano /boot/config.txt`, for example. Finally, shut the client Raspberry Pi down with `sudo poweroff`.
 
+## Ethernet MAC address
+Before configuring network boot make a note of the serial number and mac address so that the board can be identified by the TFTP/DHCP server.
+
+On Raspberry Pi 4 the MAC address is programmed at manufacture and there is no link between the MAC address and serial number. Both the MAC address and serial numbers are displayed on the bootloader HDMI diagnostics screen.
```

Comment:
You might want to mention that the HDMI diagnostics screen is only shown if the bootloader is taking longer than a certain time to boot. I believe it is 8 seconds on current versions. Someone new to Pi 4 with a Pi that has a recent EEPROM version will probably never have seen the HDMI diagnostics screen.

Reply:
I don't think we should document the HDMI diagnostics screen here but can add a link to the page that James created a while ago.